toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -55,6 +55,9 @@ const (
 	// CmdlineSELinuxEnforcingArg is the arg required for forcing SELinux to be in enforcing mode.
 	CmdlineSELinuxEnforcingArg = "enforcing=1"
 
+	// CmdlineSELinuxPermissiveArg is the arg for SELinux to be in force-permissive mode.
+	CmdlineSELinuxPermissiveArg = "enforcing=0"
```

Comment:
re [comment](https://github.com/microsoft/azurelinux/pull/10584/files#r1844721925) , now we moved to the new repo, do we still ask for permission to modify code in this file?

Reply:
Nope